Doc/library/sched.rst

```diff
@@ -19,21 +19,24 @@ scheduler:
 
    The :class:`scheduler` class defines a generic interface to scheduling events.
    It needs two functions to actually deal with the "outside world" --- *timefunc*
-   should be callable without arguments, and return  a number (the "time", in any
-   units whatsoever). If time.monotonic is not available, the *timefunc* default
-   is time.time instead. The *delayfunc* function should be callable with one
-   argument, compatible with the output of *timefunc*, and should delay that many
+   is a callable without arguments, and returns a number (the "time", in any
+   units whatsoever). The *delayfunc* function is a callable with one
+   argument, compatible with the output of *timefunc*, and delays for that many
    time units. *delayfunc* will also be called with the argument ``0`` after each
    event is run to allow other threads an opportunity to run in multi-threaded
    applications.
 
    .. versionchanged:: 3.3
-      *timefunc* and *delayfunc* parameters are optional.
+      The *timefunc* and *delayfunc* parameters are optional.
 
    .. versionchanged:: 3.3
-      :class:`scheduler` class can be safely used in multi-threaded
+      :class:`scheduler` can be safely used in multi-threaded
       environments.
 
+   .. versionchanged:: 3.5
+      *timefunc* will always default to :func:`time.monotonic`,
+      given that the function is available on all platforms.
```

Comment:
That's wrong. time.monotonic is used since Python 3.3:
https://docs.python.org/release/3.3.7/library/sched.html#sched.scheduler

Reply:
Complete the 3.3 note, or just remove this 3.5 note.

Reply:
Yes, but I'm using _always_ to denote that only as of 3.5 is `time.monotonic()` guaranteed to be what is actually used as the callable.

https://github.com/python/cpython/blob/bf9cccb2b54ad2c641ea78435a8618a6d251491e/Lib/sched.py#L38

I think it would be most straightforward to just remove the 3.5 note to avoid confusion.